WabiSabiMonitor.ApplicationCore/AnalysisStoreManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+using WabiSabiMonitor.ApplicationCore.Data;
+using WabiSabiMonitor.ApplicationCore.Interfaces;
+using WabiSabiMonitor.ApplicationCore.Utils.Bases;
+using WabiSabiMonitor.ApplicationCore.Utils.Helpers;
+using WabiSabiMonitor.ApplicationCore.Utils.WabiSabi.Models.Serialization;
+
+namespace WabiSabiMonitor.ApplicationCore
+{
+    public class AnalysisStoreManager : PeriodicRunner
+    {
+        private readonly IAnalyzer _analyzer;
+        private readonly IRoundsDataFilter _roundsDataFilter;
+        private DateTime _lastDate;
+        public Dictionary<DateTime, Analyzer.Analysis> Analysis { get; private set; }
+
+        public AnalysisStoreManager(IAnalyzer analyzer, IRoundsDataFilter roundsDataFilter, TimeSpan period) : base(period)
+        {
+            _analyzer = analyzer;
+            _roundsDataFilter = roundsDataFilter;
+            _lastDate = DateTime.Now;
+            Analysis = new();
+        }
+
+        protected override async Task ActionAsync(CancellationToken cancel)
+        {
+            var now = DateTime.Now;
```

Comment:
I didn't see that but please never use `DateTime.Now` in this project. Instead use DateTime.UtcNow